### PR TITLE
Chore/new favourite layout

### PR DESCRIPTION
**Pull request by olotintemitope**
This pull request solves the following issues;
1. The navigation sidebar on the channel page has dead links.
2. The favourite page was using an old layout.
3. Episodes without comments prints error messages  on the page.

### CODE REVIEW

File: app/Http/Controllers/CommentController.php [hunk 1]
```diff
@@ -24,20 +24,41 @@ protected function create(array $data)
...
+            'message' => 'Comment created Successfully',
+            'status_code' => 200,
+            'commentId' => $newComment->id
             ];
     }
 
     /**
+     * Get 10 comments from current episode
+     */
+    public function fetchComment(Request $request)
```
Comment (johnkariuki) [inline app/Http/Controllers/CommentController.php:42]: @andela-tolotin Please write tests to assert that  `fetchComment` returns the next comments.

**Comment by olotintemitope** [comment]
@andela-jkariuki Noted

**Comment by unicodeveloper** [comment]
When would this be ready @andela-tolotin ?

**Comment by olotintemitope** [comment]
@unicodeveloper, I have implemented the issues I listed above.

**Comment by unicodeveloper** [comment]
Please rebase to resolve merge conflicts @andela-tolotin

**Comment by olotintemitope** [comment]
@unicodeveloper I will do that as soon as possible

**Comment by olotintemitope** [comment]
@unicodeveloper I have rebased this branch and everything is up to date. You can merge it.

**Pull request merged**